test/e2e/tests/request-queuing/ui.spec.js

```diff
@@ -93,29 +96,23 @@ async function switchToNotificationPopoverValidateDetails(
   const windowHandles = await driver.getAllWindowHandles();
   await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog, windowHandles);
 
-  // Get UI details
-  const networkPill = await driver.findElement(
-    // Differs between confirmation and signature
-    '[data-testid="network-display"], [data-testid="signature-request-network-display"]',
-  );
-  const networkText = await networkPill.getText();
-  const originElement = await driver.findElement(
-    // Differs between confirmation and signature
-    '.confirm-page-container-summary__origin bdi, .request-signature__origin .chip__label',
-  );
-  const originText = await originElement.getText();
```

Comment:
this is a bad pattern that can cause flakiness: if we get the element without waiting for the exact text contents we want, it can be a race condition when the element is rendered but the value is not there yet, making all the assertions fail